internal/cmd/flags.go

```diff
@@ -16,37 +16,111 @@ package cmd
 
 import (
 	"errors"
+	"strings"
 
 	"github.com/sapcc/limesctl/internal/core"
 )
 
-// requestFilterFlags define parameters for Limes API requests.
-type requestFilterFlags struct {
-	Area     string `help:"Resource area."`
-	Service  string `help:"Service type."`
-	Resource string `help:"resource name."`
+///////////////////////////////////////////////////////////////////////////////
+// Limes' API request filter flags.
+
+type requestFilterOpts struct {
+	area     []string
+	service  []string
+	resource []string
```

Comment:
Mirroring my comment on gophercloud-sapcc, these fields should have plural names.